### ISSUE TITLE
Clarify licence situation

### ISSUE THREAD

**Issue by DavidBruant** [mozilla/readability#8]
The original code is Apache2, but I imagine additional work by Moz folks will be MPL2

**Comment by leibovic** [comment]
I think that what starts as Apache2 stays Apache2, even if it has been further developed in mozilla-central. But let's ask @gerv :)

**Comment by gerv** [comment]
This issue lacks context, but the general rule is: if Mozilla people enhance some e.g. Apache2 code, we make our changes Apache2 so they can be reincorporated into the upstream project. This applies to all permissive licenses, e.g. MIT or BSD, which are on code we get from elsewhere.

If it's our code (e.g. from Gaia), then yes, we keep it Apache2 as well.

Does that help?

**Comment by leibovic** [comment]
Thanks for the response, and I apologize for not explaining the situation more clearly.

Here's some more context: A few years ago we landed this Apache2 code (Readbility.js) in mozilla-central, and since then we have heavily modified it. The whole time we kept the original Apache2 license header in the file, and then now we've moved this file back into its own repo (this repo), so that it can be shared by other applications (e.g. Firefox for iOS).

So it sounds like the answer here is that this library is (and always has been) Apache2. We should add a LICENSE file to the repo to make that more clear.
